langkit/ocaml_api.py

```diff
@@ -328,9 +329,17 @@ def from_module(typ, value):
                 value
             )
 
-        if isinstance(type, ct.EntityType):
+        if type.is_entity_type:
             # For entity type, the unwrap function is inside the root node
-            return from_module(T.root_node, value)
+            if check_for_none:
+                fmt_check = (
+                    "match {} with Some n -> {} "
+                    + "| None -> make EntityStruct.c_type"
```

Comment:
Python allows you to have consecutive string literals: they are considered one big literal. So you could write instead:

```python
fmt_check = ('match {} with'
             ' Some n -> {}'
             ' | None -> make EntityStruct.c_type')
```
and given that the return statement is right after:

```python
return ('match {} with'
        ' Some n -> {}'
        ' | None -> make EntityStruct.c_type'
        .format(value, from_module(T.root_node, 'n'))
```